app/src/main/java/co/jbear/euphony_hub/MainActivity.kt

```diff
@@ -28,9 +27,17 @@ class MainActivity : ComponentActivity() {
         Manifest.permission.WRITE_EXTERNAL_STORAGE
     )
 
+    private val requestPermissionsLauncher =
+        registerForActivityResult(ActivityResultContracts.RequestMultiplePermissions()) { permissions ->
+            if (!permissions.entries.all { it.value }) {
+                Toast.makeText(applicationContext, "You should get permissions to use euphony-hub", Toast.LENGTH_LONG).show()
```

Comment:
Now, our app name is ```euphony-hub-collector```, so would you also change this toast message like this?:
```suggestion
                Toast.makeText(applicationContext, "You should get permissions to use euphony-hub-collector", Toast.LENGTH_LONG).show()
```

Reply:
fixed. :thumbsup: